project-management/Understanding_Swim_Lanes.md

```diff
@@ -0,0 +1,56 @@
+# Contents
+
+1.0 BACKLOG
+
+2.0 TO-DO (CURRENT SPRINT)
+
+3.0 IN DEVELOPMENT/PROGRESS
+
+4.0 BLOCKED
+
+5.0 IN PULL REQUESTS (AWAITING REVIEW)
+
+6.0 READY FOR TESTING
+
+7.0 IN TESTING
+
+8.0 READY FOR REVIEW (TESTING)
+
+9.0 DONE
+
+### 1.0	Backlog
+A list of issues created predominantly by the Product Owner (Alan Turning) which are then team review in the backlog refinement session.
```

Comment:
```suggestion
A list of issues created predominantly by the Product Owner (Turing Institute) which are then team review in the backlog refinement session.
```

Note that Alan Turing is a mathematician who died a long time ago. The preferred abbreviation of the Alan Turing Institute is either the Turing Institute or the Turing.